Echo.Process/SessionId.cs

```diff
@@ -42,48 +42,20 @@ public SessionId(string value)
         public override string ToString() =>
             Value;
 
-        public override int GetHashCode() =>
-            Value.GetHashCode();
-
         public static implicit operator SessionId(string value) =>
             new SessionId(value);
 
-        public bool Equals(SessionId other) =>
-            Value.Equals(other.Value);
-
-        public int CompareTo(SessionId other) =>
-            String.Compare(Value, other.Value, StringComparison.Ordinal);
-
-        public int CompareTo(object obj) =>
-            obj == null
-                ? -1
-                : obj is SessionId
-                    ? CompareTo((SessionId)obj)
-                    : -1;
-
-        public static bool operator == (SessionId lhs, SessionId rhs) =>
-            lhs.Equals(rhs);
-
-        public static bool operator !=(SessionId lhs, SessionId rhs) =>
-            !lhs.Equals(rhs);
-
-        public override bool Equals(object obj) =>
-            obj is SessionId
-                ? Equals((SessionId)obj)
-                : false;
     }
 
     /// <summary>
     /// Supplementary session id new type
     /// </summary>
-    public class SupplementarySessionId : NewType<SupplementarySessionId, SessionId>
+    public class SupplementarySessionId : SessionId
```

Comment:
A supplemntary session ID isn't a session ID.  They should be considered incompatible with each other, so let's not inherit from `SessionId`.

Reply:
Also, make it into a `struct` like the `SessionId`